src/sass/static/affiliates.scss

```diff
@@ -6,6 +6,15 @@
             margin-top: 3rem;
         }
     }
+    @media screen and (max-width: 480px) {
+        .sidebar-container {
```

Comment:
can we use the same styling that terms and conditions menu is using? if not then it's fine to do it separately here

Reply:
`sidebar-collapsible` (in terms and conditions) styles are not specific to mobile :)
Also I want to preserve `sidebar` desktop styles identical to the one on get-started/binary-options and others